src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSsl.cs

```diff
@@ -402,25 +399,6 @@ private static unsafe int AlpnServerSelectCallback(IntPtr ssl, out byte* outp, o
             return Ssl.SSL_TLSEXT_ERR_NOACK;
         }
 
-        private static void UpdateCAListFromRootStore(SafeSslContextHandle context)
-        {
-            using (SafeX509NameStackHandle nameStack = Crypto.NewX509NameStack())
-            {
-                //maintaining the HashSet of Certificate's issuer name to keep track of duplicates 
-                HashSet<string> issuerNameHashSet = new HashSet<string>();
-
-                //Enumerate Certificates from LocalMachine and CurrentUser root store 
-                AddX509Names(nameStack, StoreLocation.LocalMachine, issuerNameHashSet);
-                AddX509Names(nameStack, StoreLocation.CurrentUser, issuerNameHashSet);
-
-                Ssl.SslCtxSetClientCAList(context, nameStack);
-
-                // The handle ownership has been transferred into the CTX.
-                nameStack.SetHandleAsInvalid();
-            }
-
-        }
-
         private static void AddX509Names(SafeX509NameStackHandle nameStack, StoreLocation storeLocation, HashSet<string> issuerNameHashSet)
```

Comment:
Does this method still have callers?

Reply:
Pretty sure it doesn't have any more. I'll remove it as well